src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -42,15 +50,7 @@ public static RGSequenceManager GetInstance()
      */
     public void LoadSequences()
     {
-        if (_loadingSequences == false)
-        {
-            // not perfectly thread safe.. but close enough for this case
-            _loadingSequences = true;
-            new Thread(() =>
-            {
-                _loadedSequences = ResolveSequenceFiles();
-            }).Start();
-        }
+        StartCoroutine(ResolveSequenceFiles());
```

Comment:
nit: Is this LoadSequences called anywhere else (I can't tell from the GitHub PR review). If not, we should just call this code directly in start.

Reply:
it is called in multiple locations yes